clusters/app.ci/dgraph/admin_dgraph-sts.yaml

```diff
@@ -0,0 +1,106 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: dgraph-public
+  namespace: ci
+  labels:
+    app: dgraph
+spec:
+  type: ClusterIP
+  ports:
+  - port: 5080
+    targetPort: 5080
+    name: grpc-zero
+  - port: 6080
+    targetPort: 6080
+    name: http-zero
+  - port: 8080
+    targetPort: 8080
+    name: http-alpha
+  - port: 9080
+    targetPort: 9080
+    name: grpc-alpha
+  - port: 8000
+    targetPort: 8000
+    name: http-ratel
+  selector:
+    app: dgraph
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: dgraph
+  namespace: ci
+spec:
+  serviceName: "dgraph"
+  replicas: 1
+  selector:
+      matchLabels:
+        app: dgraph
+  template:
+    metadata:
+      labels:
+        app: dgraph
+    spec:
+      containers:
+      - name: ratel
+        image: dgraph/ratel:latest
```

Comment:
Should we use one of [those tags](https://hub.docker.com/r/dgraph/ratel/tags) rather than `latest`?

Reply:
This is ratel. We shouldn't care about the version. `latest` will be the best option. To be honest we don't even need it for now. We could remove it completely. WDYT?

Reply:
Is there an alternative user-friendly way to query the db and visualize the resulting graph other than using Ratel? It's a long way to go before having our own UI ready, if that's what you mean.